### PR TITLE
Replacing all azdata with az

### PR DESCRIPTION
**Pull request by candiceye**
Replaces functionality of azdata with az in the arc extension and resource-deployment extension.

**Notes**:
- Commented out the prompts for install/update of Azure CLI arcdata extension for now.
- Commented out the relevant unit tests for now.

**Main changes:**
- Added a dropdown 'infrastructure' field to the SQL MIAA deployment wizard.
- Removed all instances of (Preview) for SQL MIAA and DC.
- Removed user and password requirement for connecting a DC.
- Removed login requirements for az that were present in azdata.
- Changed SQL MIAA, Postgres, DC deployment notebooks to use updated flags.

### CODE REVIEW

File: package.json
Comment (Charles-Gagnon) [inline package.json:78]: Mistake? You shouldn't be adding stuff to the root package.json unless you're adding something to core

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb
Comment (candiceye) [reply extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb:26]: I'm using the VS Code editor but I don't see how to edit the line ending. Here's a screenshot of how I see it: 
image

Comment (Charles-Gagnon) [reply extensions/arc/notebooks/arcDeployment/deploy.arc.data.controller.ipynb:26]: You can't edit it directly in the VS Code editor after the fact - but in my testing it defaulted to using `\n` when editing cells myself.

Anyways just doing a find/replace on these files to revert them would be the easiest way to undo these changes.

File: package.json
Comment (candiceye) [reply package.json]: I'll revert this file. I was having some issues getting elliptic to work at some point. I also reverted the yarn.lock file change that contained elliptic@^6.5.4

File: extensions/arc/src/localizedConstants.ts
Comment (candiceye) [reply extensions/arc/src/localizedConstants.ts:11]: In yesterday's shiproom I was asked to remove these instances of Preview, but if that's the case then I'll add them back in. What is the process for those reviews? Would I be able to change these to non-preview right after I release this with the preview text?

Comment (candiceye) [reply extensions/arc/src/localizedConstants.ts:11]: Adding @twright-msft to this in case further discussion is needed.

Comment (Charles-Gagnon) [reply extensions/arc/src/localizedConstants.ts]: Let's chat offline about this.

The review process is still ongoing and/or hasn't started for the various parts so no, you won't be able to remove the preview tag until that is done.

File: extensions/arc/src/test/ui/dashboards/postgresConnectionStrings.test.ts
Comment (candiceye) [reply extensions/arc/src/test/ui/dashboards/postgresConnectionStrings.test.ts:1]: A lot of the tests are actually not up-to-date and not going to compile if I add .skip to the describe. I'm wondering if it's worth it then to spend the time making the tests compile if I will skip them anyways (for this release). Perhaps this is something that could be done after GA?

Comment (Charles-Gagnon) [reply extensions/arc/src/test/ui/dashboards/postgresConnectionStrings.test.ts:1]: Up to you, but yeah that's fine by me as long as all the tests pass in our pipelines

File: extensions/azcli/src/localizedConstants.ts
Comment (Charles-Gagnon) [inline extensions/azcli/src/localizedConstants.ts:29]: remove en-us from the links here

**Review by Charles-Gagnon — APPROVED** [review]
I still want to sync up with you on the addition of the Infrastructure field to resource deployment, but that can be done after this is merged in.

There does appear to be a compile error though so make sure you fix that first - but assuming that it's just a simple fix then this should be fine to merge in once that's resolved. (all checks should be passing before merging in)

**Pull request merged**
